sdk/src/main/java/com/entracer/sdk/analytics/EntracerAPI.java

```diff
@@ -87,6 +90,22 @@ public String getToken() {
         return this.mToken;
     }
 
+    /**
+     * Returns api baseUrl for entracer.
+     * @return baseUrl.
+     */
+    public String getBaseUrl() {
+        return baseUrl;
+    }
+
+    /**
+     * Sets api baseUrl for entracer.
+     * @return void.
```

Comment:
We can remove this @return documentation comment as method does not return anything.
`/**
 * @param base url for api to set
 */
public void setBaseUrl(String baseUrl)`